src/Sarif.Viewer.VisualStudio/Views/StringResources.xaml

```diff
@@ -0,0 +1,62 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <!-- SARIF Viewer Control -->
+    <system:String x:Key="SarifViewerControl_DialogHeader_Separator_Text_String">|</system:String>
```

Comment:
Does this need to be localized?

Reply:
I guess not, I just don't like having literals in markup.

Reply:
@EasyRhinoMSFT In your response to my other comment, you cited ease of localization in support of your position, which is reasonable. But then here, why should we make the localizers look at this and wonder how they're supposed to translate "`|`"? :-) IMO we should just take that off their plates.

Reply:
Fair enough